src/Serializer/SerializableInterface.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sentry\Serializer;
+
+interface SerializableInterface
```

Comment:
Add a description to this interface: `This interface can be used to customize how an object is serialized in the payload of an event.`